app/services/book_service.rb

```diff
@@ -27,13 +27,16 @@ def self.books_on_shelf(shelf, auth_token)
     JSON.parse(body, symbolize_names: true)
   end
 
-  def self.add_book(volume_id, auth_token)
-    shelf =  BookService.book_shelves(auth_token)[:items].find do |shelf|
-                shelf[:title] == "To read"
-              end
-    shelf_id = shelf[:id]
+  def self.add_book(shelf_id, volume_id, auth_token)
+    # shelf =  BookService.book_shelves(auth_token)[:items].find do |shelf|
+    #             shelf[:title] == "To read"
+    #           end
+    # shelf_id = shelf[:id]
+    conn(auth_token).get "/books/v1/mylibrary/bookshelves/#{shelf_id}/addVolume?volumeId=#{volume_id}"
```

Comment:
Love this change - It really cleans things up and I love the added functionality it provides the user. Thanks for uncovering this!